pkg/apis/monitoring/v1/types.go

```diff
@@ -545,6 +548,57 @@ type ArbitraryFSAccessThroughSMsConfig struct {
 	Deny bool `json:"deny,omitempty"`
 }
 
+type PrometheusTracingConfig struct {
+	// Client used to export the traces. Options are 'http' or 'grpc'.
+	//+kubebuilder:validation:Enum=http;grpc
+	// +optional
+	ClientType PrometheusTracingClientType `json:"clientType"`
+
+	// Endpoint to send the traces to. Should be provided in format <host>:<port>.
+	Endpoint string `json:"endpoint"`
+
+	// Sets the probability a given trace will be sampled. Must be a float from 0 through 1.
+	// +optional
+	SamplingFraction string `json:"samplingFraction"`
+
+	// If disabled, the client will use a secure connection.
+	// +optional
+	Insecure *bool `json:"insecure"`
+
+	// Key-value pairs to be used as headers associated with gRPC or HTTP requests.
+	Headers map[string]string `json:"headers"`
+
+	// Compression key for supported compression types. Supported compression: gzip
+	//+kubebuilder:validation:Enum=gzip
+	Compression PrometheusTracingCompression `json:"compression"`
+
+	// Maximum time the exporter will wait for each batch export. Default '10s'
+	// +kubebuilder:default:="10s"
```

Comment:
I would leave the default to "".

Reply:
So maybe it could be `//+optional` instead? Is there any difference?